site/content/en/docs/Reference/api.md

```diff
@@ -605,6 +737,7 @@ does not match:
 | double_arg | [string](#string) |  | Name of the ticket&#39;s search_fields.double_args this Filter operates on. |
 | max | [double](#double) |  | Maximum value. |
 | min | [double](#double) |  | Minimum value. |
+| exclude | [DoubleRangeFilter.Exclude](#openmatch.DoubleRangeFilter.Exclude) |  | Which bounds would be excluded when comparing with a ticket&#39;s search_fields.double_args value. BETA FEATURE WARNING: This field and the associated values are not finalized and still subject to possible change or removal. |
```

Comment:
"Which bounds would be excluded when comparing with a ticket"

Probably it's better to rephrase it somehow?

Reply:
`Defines the bounds to apply when filtering tickets by their search_fields.double_args value` what do you think??